gatsby-browser.tsx

```diff
@@ -42,7 +43,7 @@ export const wrapPageElement: GatsbyBrowser<
 
   // client side redirect on paths that don't have a locale in them. Most useful
   // on dev env where we don't have server redirects
-  if (!isLang(pathLocale)) {
+  if (IS_DEV && !isLang(pathLocale)) {
```

Comment:
On prod we have server side redirects. We don't need this. In dev is handy to avoid receiving a 404 page at `/`.